mailpoet/assets/js/src/email-editor/integrations/mailpoet/components/send-preview-email.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button, Modal, TextControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import ReactStringReplace from 'react-string-replace';
+
+type SendPreviewEmailProps = {
+  isOpen: boolean;
+  closeCallback: () => void;
+};
+
+export function SendPreviewEmail({
+  isOpen,
+  closeCallback,
+}: SendPreviewEmailProps) {
+  let description = ReactStringReplace(
+    __(
+      'Send yourself a test email to test how your email would look like in different email apps. You could also enter your [link1]Mail Tester[/link1] email below to test your spam score. [link2]Learn more[/link2].',
+      'mailpoet',
+    ),
+    /\[link1\](.*?)\[\/link1\]/g,
+    (match, i) => (
+      <a
+        key={i}
+        href="https://www.mail-tester.com/"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        {match}
+      </a>
+    ),
+  );
+  description = ReactStringReplace(
+    description,
+    /\[link2\](.*?)\[\/link2\]/g,
+    (match, i) => (
+      <a
+        key={i}
+        href="https://kb.mailpoet.com/article/147-test-your-spam-score-with-mail-tester"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        {match}
+      </a>
+    ),
+  );
```

Comment:
Please, let's not use ReactStringReplace. We can use `createInterpolateElement` please see https://mailpoetdev.wordpress.com/2023/10/04/plugins-now-use-wp-to-enqueue-js-and-css/

Reply:
The power of habit. I'll fix it. Thank you
@costasovo Should I also fix it in [the details panel](https://github.com/mailpoet/mailpoet/blob/c8fc6bf49aa8add3f989530c792ae1cf5fac31ae/mailpoet/assets/js/src/email-editor/integrations/mailpoet/components/settings-panel/details-panel.tsx#L38-L68)? Or will it be a part of the custom editor bootstrap?

Reply:
We can do it in the custom editor bootstrap. Good idea! 👍